src/SprykerEco/Yves/Computop/ComputopFactory.php

```diff
@@ -140,6 +145,14 @@ public function createEasyCreditForm()
         return new EasyCreditSubForm();
     }
 
+    /**
+     * @return \SprykerEco\Yves\Computop\Form\PayuCeeSingleSubForm
+     */
+    public function createPayUCeeSingleForm(): PayuCeeSingleSubForm
```

Comment:
* Rename the method to `createPayuCeeSingleSubForm()`.

Reply:
Done